internal/lifecycle/utils.go

```diff
@@ -0,0 +1,29 @@
+package lifecycle
+
+import (
+	"github.com/buildpacks/lifecycle/buildpack"
+)
+
+func PrependExtensions(orderBp buildpack.Order, orderExt buildpack.Order) buildpack.Order {
```

Comment:
The `utils` file name threw up a flag for me. As far as GitHub could see, this function is only used in detector.go...maybe just move it there? Or if it is worth splitting into its own file, perhaps a more descriptive name?

Reply:
Makes sense - I'll move it.